src/libraries/Microsoft.PowerFx.Interpreter/ParsedExpression.cs

```diff
@@ -139,6 +141,12 @@ public async Task<FormulaValue> EvalAsync(CancellationToken cancellationToken, I
                 ServiceProvider = new BasicServiceProvider(runtimeConfig?.ServiceProvider, hasInnerServices ? innerServices : null)
             };
 
+            if (_features != null)
+            {
+                innerServices.AddService(_features);
+                hasInnerServices = true;
+            }

```

Comment:
The local `hasInnerServices` is only used 6 lines above; no need to set it again